images/bootstrap/Dockerfile

```diff
@@ -116,6 +116,8 @@ ADD ["./barnacle/barnacle", "/usr/local/bin"]
 # END: DOCKER IN DOCKER SETUP
 #
 
+# Cleanup unnecessary package(s)
+RUN apt-get remove -y systemd
```

Comment:
this should be early in the image since it's unlikely to change any more often than the base image.

Reply:
Moved it as early as i can. any earlier and it fails with the following

```
The following packages will be REMOVED:
  init systemd systemd-sysv
WARNING: The following essential packages will be removed.
This should NOT be done unless you know exactly what you are doing!
  init systemd-sysv (due to init)
0 upgraded, 0 newly installed, 3 to remove and 0 not upgraded.
After this operation, 11.8 MB disk space will be freed.
E: There are problems and -y was used without --force-yes
The command '/bin/sh -c apt-get remove -y systemd' returned a non-zero code: 100
make: *** [build] Error 100
```